bitnami/wordpress/README.md

```diff
@@ -513,6 +513,10 @@ To enable the new features, it is not possible to do it by upgrading an existing
 
 ## Upgrading
 
+### To 18.0.0
+
+This major release bumps the MariaDB version to 11.1. No major issues are expected during the upgrade.
```

Comment:
Can we confirm so? Did you run manual tests to double-check it?

Reply:
Yes, I run locally, and everything going well.